### PR TITLE
feat(gao-de-location): add plugin

### PR DESCRIPTION
**Pull request by waliu**
Through this plug-in, you can get the local location. Reverse Address Coding

### CODE REVIEW

File: src/@ionic-native/plugins/gao-de-location/index.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  platforms: ['Android', 'iOS'] // Array of platforms supported, example: ['Android', 'iOS']
+})
+@Injectable()
+export class GaoDeLocation extends IonicNativePlugin {
+  /**
+   * Get the current location
+   * @returns {Promise<any>} Returns a Promise that resolves with the success return, or rejects with an error.
+   */
+  @Cordova()
+  getCurrentPosition(): Promise<any> {
```
Comment (danielsogl) [inline src/@ionic-native/plugins/gao-de-location/index.ts]: Please provide the correct typings as an interface.

File: src/@ionic-native/plugins/gao-de-location/index.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ *
+ * this.gaoDeLocation.getCurrentPosition()
+ *   .then((res: any) => console.log(res))
+ *   .catch((error: any) => console.error(error));
+ *
+ * ```
+ */
+@Plugin({
+  pluginName: 'GaoDeLocation',
+  plugin: 'cordova-plugin-gaodelocation-chenyu', // npm package name, example: cordova-plugin-camera
```
Comment (danielsogl) [inline src/@ionic-native/plugins/gao-de-location/index.ts]: Remove the placeholder comments

File: src/@ionic-native/plugins/gao-de-location/index.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+ * ```typescript
+ * import { GaoDeLocation } from '@ionic-native/gao-de-location';
+ *
+ *
+ * constructor(private gaoDeLocation: GaoDeLocation) { }
+ *
+ * ...
+ *
+ *
+ * this.gaoDeLocation.getCurrentPosition()
```
Comment (danielsogl) [inline src/@ionic-native/plugins/gao-de-location/index.ts]: Add the correct interface as response type and remove `any`

File: src/@ionic-native/plugins/gao-de-location/index.ts [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { Cordova, IonicNativePlugin, Plugin } from '@ionic-native/core';
+
+/**
+ * @name Gao De Location
+ * @description
+ * Get the current location
```
Comment (danielsogl) [inline src/@ionic-native/plugins/gao-de-location/index.ts]: Please provide a better description. I'm not that familiar with Chinese so I don't know the difference between this plugin and the Cordova geolocation plugin.

**Comment by waliu** [comment]
Do you need any other changes?

**Comment by danielsogl** [comment]
No its fine. I will merge all open PR's tomorrow.

**Pull request merged**